README.md

```diff
@@ -18,7 +18,7 @@ To use `http-body`, first add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-http-body = "0.1"
+http-body = "0.1.0"
```

Comment:
This isn't needed, is it?

Reply:
Technically... but I like it :) It explicitly sets `0.1.0` as the minimum instead of "forgetting" the patch :shrug: I can remove it if you don't like it.

Reply:
I'm going to move forward. We can always change it later.